uber/site_sections/schedule.py

```diff
@@ -105,6 +106,40 @@ def schedule_tsv(self, session):
             'schedule': sorted(schedule.items(), key=lambda tup: c.ORDERED_EVENT_LOCS.index(tup[1][0]['location']))
         })
 
+    def ical(self, session, **params):
+        icalendar = ics.Calendar()
+
+        if 'locations' not in params or not params['locations']:
```

Comment:
Just to clarify, this handles two situations:
A GET request with no `locations` parameter (e.g., `/schedule/ical`)
A POST request with a `locations` parameter that doesn't have a value, which is what happens if you press the button on the internal schedule without filtering any locations down first.